addons/l10n_pe_website_sale/demo/website_demo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+
+    <record id="default_website_pe" model="website">
+        <field name="name">(PE) Peruvian Website</field>
+        <field name="company_id" ref="l10n_pe.demo_company_pe" />
+        <field name="logo" type="base64" file="l10n_pe_website_sale/static/description/icon.png" />
+        <field
+            name="domain"
+            model="ir.config_parameter"
+            eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"
+        />
+    </record>
+    <function model="payment.provider" name="copy">
+        <value eval="[ref('payment.payment_provider_transfer')]" />
+        <value eval="{'company_id': ref('l10n_pe.demo_company_pe'), 'state': 'enabled'}" />
+    </function>
+    <function model="product.pricelist" name="write">
```

Comment:
Hello @JulienVR I think it is not that we need a pricelist, rather than that, we're creating one for Peru, otherwise, you need to configure one of the existing ones to be the default pricelist for the e-commerce. Right @desdelinux ?

Reply:
I'm still unsure about the pricelist (and it is empty as seen in the screenshot), also since it's in the demo data, a real user will have to configure its own pricelist anyway, if that's what you meant. 
![image](https://github.com/odoo/odoo/assets/60702485/cff8ec4c-0fbc-4a6f-82e1-dd24e7657582)

Reply:
Ok, let's delete the price list

@AndreaGeraldo